barretenberg/cpp/src/barretenberg/bb/main.cpp

```diff
@@ -199,6 +200,22 @@ void writeVk(const std::string& bytecodePath, const std::string& outputPath)
     }
 }
 
+void writePk(const std::string& bytecodePath, const std::string& outputPath)
```

Comment:
write_pk more idiomatic

Reply:
ah the rest of the file is like this, probably fine and fixing this is all or nothing